server/src/middlewares/auth.js

```diff
@@ -34,21 +34,29 @@ const checkJwt = async (req, res, next, prisma) => {
       }`
     )
 
+  const speId = conf.tagCategories[0].labels[1].id
```

Comment:
Pas besoin d’abréger :)
```suggestion
  const specialityId = conf.tagCategories[0].labels[1].id
```